src/applications/appeals/995/content/reviewErrors.js

```diff
@@ -12,14 +15,14 @@ const reviewErrors = {
       ) {
         return { chapterKey: 'evidence', pageKey: 'evidenceSummary' };
       }
-      if (err.startsWith('veteran') || err.includes(PRIMARY_PHONE)) {
```

Comment:
What's the story behind the `veteran` check?

Reply:
In the form data, all the contact info is wrapped inside a `veteran` object:

```js
veteran: {
  address: { /* ... */ },
  phone: { /* ... */ },
  email: '',
},
```

So when validation of the form happens, we get back a processed error message array - saved to Redux in `form.formErrors.errors` that looks like this:

```jsonc
{
  errors: [
    {
      name: 'country',
      index: null,
      message: 'Veteran.address country',
      chapterKey: '',
      pageKey: 'confirmContactInfo'
    },
    {
      name: 'street',
      index: null,
      message: 'Veteran.address street',
      chapterKey: '',
      pageKey: 'confirmContactInfo'
    },
    {
      name: 'postalCode',
      index: null,
      message: 'Veteran.address postalCode',
      chapterKey: '',
      pageKey: 'confirmContactInfo'
    },
    {
      name: 'veteran.email',
      index: null,
      message: 'Veteran.email does not conform to the email format',
      chapterKey: '',
      pageKey: 'confirmContactInfo'
    }
  ]
}
```

As you can see, since we're using a Custom page, the `chapterKey` and/or the `pageKey` may be undefined. That's why a `reviewErrors` override was added to check the name (path to data) and return a `chapterKey` and `pageKey` for these pages. So in the `profileReviewErrorOverride` function, `veteran` has been replaced by the `wrapperKey` value (used by the profile definition to create this wrapper object)